blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java

```diff
@@ -27,7 +27,7 @@
 
     // todo: these should be instances per client.
     protected final Timer searchTimer = Metrics.timer(getClass(), "Search Duration");
-    protected final Timer getNextTokensTimer = Metrics.timer(getClass(), "getNextTokens Duration");
+    protected final Timer getNextTokensTimer = Metrics.timer(getClass(), "getNextTokens ES metric index query Duration");
```

Comment:
NIT:  that's a crazy name, sure you want it like that?  
Also I think the other metric labels are capitalized each word except for minor words.